### PR TITLE
For Labeling

### PR DESCRIPTION
**Pull request by ghost**
Changes of requirement in setup
Changes in Qt_main and canvas to show ON/OFF legend

### CODE REVIEW

File: config/DatabaseInfo.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import os
```
Comment (thomaskuestner) [inline config/DatabaseInfo.py]: file is redundant to /DatabaseInfo.py -> link to this one!

File: config/MRData.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class MRData:
```
Comment (thomaskuestner) [inline config/MRData.py]: redundant to /MRData.py -> link to this one!

File: GUI/PyQt/config/DatabaseInfo.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import csv
```
Comment (thomaskuestner) [inline GUI/PyQt/config/DatabaseInfo.py]: Please ensure that all modifications to in GUI/PyQt/config/DatabaseInfo.py still allow correct processing with /DatabaseInfo.py --> I will delete DatabaseInfo.py in a next commit and link then to these files!

Comment (thomaskuestner) [reply GUI/PyQt/config/DatabaseInfo.py]: Same applies to GUI/PyQt/config/MRData.py

File: config/param.yml [hunk 4]
```diff
@@ -5,7 +5,6 @@ motion_head: &id001
    labelref  : [0]
```
Comment (thomaskuestner) [inline config/param.yml]: Please leave this root file untouched -> you have the same file in PyQt/config 
Do you need to modify this one? If yes, rename it to param_GUI.yml

File: GUI/PyQt/setup.py [hunk 5]
```diff
@@ -25,9 +25,9 @@ def readme():
           tests_require=['pytest'],
```
Comment (thomaskuestner) [inline GUI/PyQt/setup.py]: Can you please create a second setup file with the current package versions that work for you.
I know for example that numpy 1.16 and sklearn have some unresolved conflicts -> GUI will not open with newest packages

**Review by thomaskuestner — CHANGES_REQUESTED** [review]
Please also delete the files "config/dump*.txt"

**Comment by thomaskuestner** [comment]
Do we still need `DatabaseInfo.py` and `MRData.py` in the GUI?
Can we decouple it?

**Review by thomaskuestner — COMMENTED** [review]
For the time being, it is ok to keep the networks as copies in the GUI/networks folder.
Please track this issue that you allow in the GUI settings to define the networks folder

-> we need to minimize duplicates!

**Review by thomaskuestner — APPROVED** [review]

**Comment by yiyi1zhang** [comment]
I have changed the import files in networks.

**Pull request merged**